test/check_for_unsafe_html.rb

```diff
@@ -17,7 +17,7 @@ def check_for_unsafe_html!
     str.gsub!(/<!\[CDATA\[.*?\]\]>/mu, "")
     return unless str =~ /&lt;[a-z]+|&amp;[#\w]+;/i
 
-    msg = "..." + $`[-200..-1] + "***HERE***" + $& + $'[0..200] + "..."
+    msg = "..." + $`[-200..] + "***HERE***" + $& + $'[0..200] + "..."
```

Comment:
Fixes offense of cop added by RuboCop 0.83 (Style/SlicingWithRange).
See https://github.com/JoeCohen/mushroom-observer/commit/36d4982